src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -4556,6 +4583,7 @@ export interface GoCheat {
   destroyNode(
     x: number,
     y: number,
+    playAsWhite?: boolean,
```

Comment:
Optional: It probably makes more sense to do this here and wherever you can.
```suggestion
    playAsWhite = false,
```